### PR TITLE
Update New-MsmqMessage.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Missing description for one of the parameter has been added now.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix  Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @tseward

Peer review needed for this PR. Thanks.

File: docset/winserver2012r2-ps/msmq/New-MsmqMessage.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ PS C:\>New-MsmqMessage -Body "Test Message" -AdminQueue ".\private$\adminqueue"
 ## PARAMETERS
 
 ### -AcknowledgeType
-{{Fill AcknowledgeType Description}}
+Specifies the type of acknowledgements supported by Messaging Queue.
```
Comment (nenonix) [inline docset/winserver2012r2-ps/msmq/New-MsmqMessage.md]:
```suggestion
Specifies the type of acknowledgement messages that can be returned by Messaging Queuing
```

**Comment by e0i** [comment]
> I suggest that a description should be more descriptive than the title already indicates. Leaving as a suggestion.

> Could you apply the same description for 2016 and 2012 versions? Thanks!

cc @sowjanya365

**Comment by tseward** [comment]
I like @nenonix's suggestion on this one.

**Comment by sowjanya365** [comment]
> Could you apply the same description for 2016 and 2012 versions? Thanks!

This is done. Kindly review. Thank you! :)

**Comment by dariomws306** [comment]
I think 2012 (not R2) version is still missing.

**Comment by sowjanya365** [comment]
> I think 2012 (not R2) version is still missing.

Yes, As there are no {{ }} brackets so I missed to add. :) 

But earlier I added in 2012 R2 and 2016. Both the articles needed that change. Now I'll update in 2012 as well. Thank you!

**Pull request merged**